static/js/realm_logo.js

```diff
@@ -5,10 +5,12 @@ const settings_config = require("./settings_config");
 exports.build_realm_logo_widget = function (is_night) {
     let logo_section_id = "#realm-day-logo-upload-widget";
     let logo_source = page_params.realm_logo_source;
+    let night_param = false;
```

Comment:
`let night_param = is_night`

Or, just call the argument `night_param`.

Reply:
Oh, I've missed that! Sorry, didn't see that it was receiving that parameter.